contracts/src/structs/Subnet.sol

```diff
@@ -143,6 +143,17 @@ struct IPCAddress {
     FvmAddress rawAddress;
 }
 
+/// @notice Validator struct for gateway constructor.
+struct GenesisValidator {
```

Comment:
Wasn't there already a type for this? Isn't this what the constructor passes?

Reply:
This is redundant with `ValidatorInfo`. It just contains the address, which we could extend the `ValidatorInfo` to contain too if needed.